AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -29,6 +29,7 @@ final class EventHub {
     private var eventNumberCounter = AtomicCounter()
     private let eventQueue = OperationOrderer<Event>("EventHub")
     private var preprocessors = ThreadSafeArray<EventPreprocessor>(identifier: "com.adobe.eventHub.preprocessors.queue")
+    private var started = false // true if the `EventHub` is started, false otherwise. Should only be accessed from within the `eventHubQueue`
```

Comment:
@shalehaha This is a bit lightweight... should I instead introduce an `AtomicBool` or would that be overkill?

Reply:
seems good to me